jdk/make/closed/autoconf/custom-hook.m4

```diff
@@ -161,6 +161,10 @@ AC_DEFUN_ONCE([OPENJ9_PLATFORM_SETUP],
       OPENJ9_PLATFORM_CODE=wa64
       if test "x$OPENJ9_LIBS_SUBDIR" = xdefault; then
         OPENJ9_BUILDSPEC="win_x86-64"
```

Comment:
Can you move this to an `else` branch so `OPENJ9_BUILDSPEC` is set exactly once?

Reply:
Already Fixed.